### PR TITLE
#4 webkit first

### PR DESCRIPTION
**Pull request by SergeyGor85**

### CODE REVIEW

**Comment by xnf** [comment]
Here is a little tip on how to write a git commit messages :) https://chris.beams.io/posts/git-commit/

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
```
Comment (xnf) [inline dist/index.html]: remove this file from the git and place `./dist` in the `.gitignore`

**Review by xnf — COMMENTED** [review]
1. remove the `dist` folder from git and put it to `.gitignore`. it is for local build only (you, me or CI)
2. format the scss file to be more ... scss-y :) nested styles, variables, etc.

File: dist/index.html (hunk 1 above)
Comment (SergeyGor85) [reply dist/index.html]: Done

File: src/index.html [hunk 2]
```diff
@@ -206,5 +206,5 @@ <h3>Contacts</h3>
             </article>
         </div>
     </footer>
-</body>
+<script src="main.js"></script></body>
```
Comment (xnf) [inline src/index.html]: There should be a way fo the webpack to add this itself during the build

**Review by xnf — APPROVED** [review]

**Pull request merged**